apps/core/scenes/search/Datepickers.js

```diff
@@ -124,7 +125,7 @@ class Datepickers extends React.Component<Props, State> {
             />
           )}
         </PickersWrapper>
-        <DatePicker
+        <RangeDatePicker
```

Comment:
As mentioned in description, there is something really hungry happening during Picker opening.
I've checked it in Chrome (so it doesn't seems to be platform specific) and as you can see it generates peak on every opening. Then it takes quite a long time (400ms) to actually show picker modal.
<img width="1030" alt="Screenshot 2019-03-25 at 11 36 51" src="https://user-images.githubusercontent.com/2660330/54913382-e73b8300-4ef2-11e9-9a83-608a68cf7281.png">